sql/qaqc/qaqc_app_units.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE IF EXISTS qaqc_app_units;
+WITH 
+classa_net_mismatch AS (
+    SELECT job_number
+    FROM FINAL_devdb
+    WHERE
+    classa_init IS NOT NULL
+    AND classa_prop IS NOT NULL
+    AND classa_net <> classa_prop - classa_init
+)
+SELECT a.job_number,
+    (CASE 
+	 	WHEN a.job_number IN (SELECT job_number FROM classa_net_mismatch) THEN 1
+	 	ELSE 0
+	END) as classa_net_mismatch
+INTO qaqc_app_units
+FROM FINAL_qaqc a;
```

Comment:
Do we need to get job numbers from `FINAL_qaqc` here? Why not just get job numbers from `FINAL_devdb`? In my understanding all the info in `qaqc_app_units` is derived from `FINAL_devdb` so it makes sense for the query to just point to that table. If that understanding is off we should talk about it more before going forward

Reply:
nah that's fine we can just use final_devdb